packages/cozy-realtime/src/index.spec.js

```diff
@@ -0,0 +1,245 @@
+import CozyRealtime, {
+  generateKey,
+  getWebSocketUrl,
+  getWebSocketToken
+} from '.'
+import { Server } from 'mock-socket'
+import MicroEE from 'microee'
+
+const COZY_URL = 'http://cozy.tools:8888'
+const WS_URL = 'ws://cozy.tools:8888/realtime/'
+const COZY_TOKEN = 'zvNpzsHILcXpnDBlUfmAqVuEEuyWvPYn'
+class CozyClient {
+  stackClient = {
+    uri: COZY_URL,
+    token: {
+      token: COZY_TOKEN
+    }
+  }
+}
+MicroEE.mixin(CozyClient)
+const cozyClient = new CozyClient()
+const pause = time => new Promise(resolve => setTimeout(resolve, time))
+
+describe('CozyRealtime', () => {
```

Comment:
Some tests have disappeared from #355:
* [should receive updated document with given id](https://github.com/cozy/cozy-libs/pull/355/files#diff-1ed3a597448c7d4e7ff049ff76b6e801R379) (asserts that both handlers for type and ids are called, once time each
* [should stop receiving created documents with given doctype](https://github.com/cozy/cozy-libs/pull/355/files#diff-1ed3a597448c7d4e7ff049ff76b6e801R442)

Some real life tests with several subscribe/unsubscribe have been lost, event if they are pretty simples.

Reply:
J'ai ajouté plus de test dans ce commit https://github.com/cozy/cozy-libs/pull/424/commits/7ceca7bb4ba3ed3b71a24140ff0b1e48ac2c83c0